### PR TITLE
Install fbsimctl to prefix directory

### PR DESCRIPTION
**Pull request by plu**

### CODE REVIEW

**Comment by plu** [comment]
See also:

* facebook/FBSimulatorControl#301
* facebook/FBSimulatorControl#302

File: fbsimctl.rb [hunk 1]
```diff
@@ -4,13 +4,12 @@ class Fbsimctl < Formula
   url "https://github.com/facebook/FBSimulatorControl/tarball/v0.1.1"
   sha256 "f971c28898457cc21d48542ddf6d269ee931190d702fe07083f4d2cddcb11fc4"
```
Comment (lawrencelomax) [inline fbsimctl.rb]: Once we've landed the PRs to fix this in the `FBSimulatorControl`, tag a new version and then update the hash for that tagged version appropriately.

Comment (plu) [inline fbsimctl.rb]: Sounds good 👍

**Comment by plu** [comment]
`git tag v0.1.2 565899f4f17df14374e430acaf624162d59e58e7`

**Comment by lawrencelomax** [comment]
The sha256 of the `v0.1.2` tarball is `439d7d03096671d6b1533dc4bf56b22b45772803bfa64060c2869cb21f65ead3`

**Comment by plu** [comment]
Just tested again locally, works find 👍 . Sorry for the wrong checksum :(.

**Comment by lawrencelomax** [comment]
cc @bhamiltoncx @k21 @coneko for the merge.

**Comment by lawrencelomax** [comment]
Thanks @k21!

**Comment by plu** [comment]
Thank you!

**Pull request merged**